app/views/activities/_activity.html.slim

```diff
@@ -16,8 +16,14 @@
     .pull-left.name
       = activity.name
     .pull-right.icon
-      - if current_user.completed_activity?(activity)
-          i.fa.fa-check-square-o
-      - else
-          i.fa.fa-square-o
+      -if current_user.is_a? Student
+        - if current_user.completed_activity?(activity)
+            i.fa.fa-check-square-o
+        - else
+            i.fa.fa-square-o
+      -else
+        div.completed_activities  
```

Comment:
In slim, the default element is div, so you can just write `.completed-activities`.